contracts/monitoring/src/msg.rs

```diff
@@ -15,11 +15,14 @@ pub enum ExecuteMsg {
         chain_name: ChainName,
         new_prover_addr: Addr,
     },
+    RegisterActiveWorkerSet {
```

Comment:
Let's use verifiers instead of workers. workers was the old term, before we started using verifiers.

Reply:
Changed it to `SetActiveVerifiers`.